staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -196,6 +196,18 @@ var (
 		[]string{"verb", "resource", "subresource"},
 	)
 
+	requestFilterLatency = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name: "apiserver_request_filter_duration",
```

Comment:
What units are these in? Should that go in the name? Or at least the description?